### PR TITLE
commented out never used variable

### PR DESCRIPTION
**Pull request by timojuez**

### CODE REVIEW

File: Pages/js/options.js [hunk 1]
```diff
@@ -411,7 +411,7 @@ function removeUrlInputs (event) {
   if(target.className.indexOf('remove_whitelist') >= 0) {
 
     var domainToRemove = target.getAttribute("data-value-to-remove");
-    var CSVToRemove = target.getAttribute("data-value-to-remove");
+    //var CSVToRemove = target.getAttribute("data-value-to-remove");
```
Comment (smcgregor) [inline Pages/js/options.js]: If it isn't used, you should just remove it.

Comment (timojuez) [inline Pages/js/options.js]: yeah, why not.

2015-03-25 0:08 GMT+02:00 Sean McGregor <notifications@github.com>:

**Comment by smcgregor** [comment]
Please keep adding tests and refactoring style on their own PRs. The pull request title is also no longer accurate.

**Comment by smcgregor** [comment]
Closing due to inactivity. Feel free to reopen.

**Pull request closed without merging**